app/src/main/java/com/conupods/Settings/SettingsInfoActivity.java

```diff
@@ -0,0 +1,35 @@
+package com.conupods.Settings;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.Button;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.conupods.OutdoorMaps.View.MapsActivity;
+import com.conupods.R;
+
+public class SettingsInfoActivity extends AppCompatActivity {
+
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.settings_page_info);
+
+        Button done = findViewById(R.id.done3);
```

Comment:
declare variables only at the beginning of blocks. (beginning of method in this case)